.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
```

Comment:
Per that blog post we actually should be getting this by default since this is a new organization